parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -672,6 +672,17 @@ static char* dump_local_assignments( void** elem, void* arg )
     if( dos > 0 ) {
         string_arena_init(info->sa);
         string_arena_add_string(info->sa, "const int %s = %s%s.value;", def->name, info->holder, def->name);
+#if 0
```

Comment:
This is a leftover from another patch allowing typed locals. Will bring it in at a later date, meanwhile I will need to remove this code.